string[] object_ids
geometry_msgs/Pose[] object_poses
float64[] object_scores